gdp/small_batch/gdp_small_batch.py

```diff
@@ -140,33 +326,42 @@ def external_ref(m, x, logic_expr=None):
     p = 0
     for j in m.j:
         ext_var[j] = x[p]
-        p = p+1
+        p = p + 1
 
     for k in m.k:
         for j in m.j:
             if k == ext_var[j]:
                 m.Y[k, j].fix(True)
-                m.Y_exists[k, j].indicator_var.fix(
-                    True)  # IS THIS REQUIRED????
-                m.Y_not_exists[k, j].indicator_var.fix(
-                    False)  # IS THIS REQUIRED????
+                m.Y_exists[k, j].indicator_var.fix(True)  # IS THIS REQUIRED????
+                m.Y_not_exists[k, j].indicator_var.fix(False)  # IS THIS REQUIRED????
             else:
                 m.Y[k, j].fix(False)
-                m.Y_exists[k, j].indicator_var.fix(
-                    False)  # IS THIS REQUIRED????
-                m.Y_not_exists[k, j].indicator_var.fix(
-                    True)  # IS THIS REQUIRED????
+                m.Y_exists[k, j].indicator_var.fix(False)  # IS THIS REQUIRED????
+                m.Y_not_exists[k, j].indicator_var.fix(True)  # IS THIS REQUIRED????
```

Comment:
Are you able to respond to this question?

Reply:
According to the disjunction part in the build model function. m.Y_exist[k, j] and m.Y_not_exist[k, j] have the corresponding algebraic constraint for activation of the coefficient value.

Reply:
So, is it required?

Reply:
I think it is required. We are trying to get the Boolean Variables, not the Binary variables.